### ISSUE TITLE
Admin fieldsets error on Django 2.1

### ISSUE THREAD

**Issue by vdboor** [mbourqui/django-publications-bootstrap#53]
Django 2.1 no longer allows to use fields multiple times in the fieldsets. Running `manage.py check` will raise an error for this.

**Comment by mbourqui** [comment]
Thanks @vdboor for reporting this issue. This seems annoying, as the fieldsets are used as visual grouping of the fields specific to some types of publication. This implies to rethink the logic of the admin, or find an alternative. If you already have an idea, feel free to open a PR ;)

**Comment by vdboor** [comment]
Yes in some way it is. The current approach however doesn't really work as that overrides field values in the next fieldset. In my local copy, I deduplicated the fieldsets. I'm not sure of the ordering through, my library is not so diverse to make a good judgement on this:

```
    fieldsets = (
        (None, {
            'fields': ('type', 'title', 'authors', 'year', 'month', 'status', 'external')}),
        ('Journal', {
            'classes': ('collapse',),
            'fields': ('journal', 'volume', 'number', 'pages')}),
        ('Technical Report', {
            'classes': ('collapse',),
            'fields': ('institution',)}),
        ('Book/Manual/Collection/Thesis', {
            'classes': ('collapse',),
            'fields': (
                'publisher', 'organization', 'school', 'location', 'country',
                'book_title', 'editor', 'chapter', 'series', 'edition'
            )}),
        ('References', {
            'fields': ('citekey', 'tags', 'code', 'url', 'doi', 'isbn')}),
        ('Description', {
            'fields': ('abstract', 'note')}),
        ('Media', {
            'classes': ('collapse',),
            'fields': ('pdf', 'image', 'thumbnail')}),
    )
```

Perhaps it would be better to categorize the fields based on their similarity instead of publication type.

**Comment by mbourqui** [comment]
You are right, there are also some issues with the current implementation.

Indeed, as you suggest, it would make sense to categorize the fields by similarity.

**Comment by vdboor** [comment]
Hi @mbourqui, could you release this change in a new version? I'd love to upgrade my site to Django 2.1

**Comment by mbourqui** [comment]
Thanks for reaching back!
The branch is not yet stable (all tests are actually failing), such that I can not merge it. I need a bit more time to fix the tests. Only then I'll be able to package a new release.
